src/stores/api/types/ethereum.go

```diff
@@ -3,11 +3,20 @@ package types
 import (
 	"time"
 
+	"github.com/ethereum/go-ethereum/core/types"
+
 	"github.com/ethereum/go-ethereum/common"
 
 	"github.com/ethereum/go-ethereum/common/hexutil"
 )
 
+// Transaction types
+const (
+	LegacyTxType     = "LEGACY"
```

Comment:
minor: Did you copy those string values from other APIs as standard? If not, I would be more incline for using lower case values as it appears to be more conventional in other enum values we have, for instance PrivateTxType

Reply:
replaced with lower case values